scheduler/pkg/kafka/gateway/infer.go

```diff
@@ -216,16 +218,57 @@ func (kc *InferKafkaHandler) createTopics(topicNames []string) error {
 		return err
 	}
 
+	// Wait for topic creation
+	logFailure := func(err error, delay time.Duration) {
+		logger.WithError(err).Errorf("waiting for topic creation")
+	}
+
+	logger.Infof("waiting for topic creation...")
+	retryPolicy := backoff.WithMaxRetries(backoff.NewConstantBackOff(1*time.Second), 60)
+	err = backoff.RetryNotify(
+		func() error {
+			return kc.ensureTopicsExist(topicNames)
```

Comment:
do you want to add logging?

Reply:
logging exists, the passed `logFailure` function is called after each failed attempt.